app/containers/EditProfilePage/renderLocationField.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LOCATION_FIELD } from 'containers/Profile/constants';
+
+import LocationList from './LocationList';
+
+const cities = (list, sendProps) =>
+  list.map(item => (
+    <li
+      role="presentation"
+      key={item.geonameId}
+      onClick={() =>
+        sendProps.chooseLocation(
+          item.geonameId,
+          `${item.name}, ${item.countryName}`,
+        )
+      }
+    >{`${item.name}, ${item.countryName}`}</li>
+  ));
+
+/* eslint-disable */
```

Comment:
What is the reason to disable eslint for this code?

Reply:
removed